src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,40 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import { UserInfo } from '../UserInfo';
+
+type Users = {
+  id: number;
+  name: string;
+  username: string;
+  email: string;
+};
+
+type Todo = {
+  id: number;
+  title: string;
+  completed: boolean
+  userId: number;
+};
+
+type TodoInfoProps = {
+  todo: Todo;
+  users: Users[];
+};
+
+export const TodoInfo: React.FC<TodoInfoProps> = ({ todo, users }) => {
+  const {
+    id,
+    title,
+    completed,
+    userId,
+  } = todo;
+  const findUser = users.find((user) => user.id === userId);
```

Comment:
findUser it sound as a function, bug it's a variable
So, make sense to rename it to the foundUser